src/django_components/perfutil/component.py

```diff
@@ -69,12 +100,15 @@
 def component_post_render(
     renderer: ComponentRenderer,
     render_id: str,
+    component_name: str,
     parent_id: Optional[str],
+    on_component_rendered: Callable[[str], None],
+    on_html_rendered: Callable[[str], str],
```

Comment:
This is the flow that smartly renders components top-down. This is where there's the queue that allows to render the components of arbitrary depth, avoiding recursion.

In this PR it was updated to also capture the component path, so show that path to the component when an error occurs during the actual rendering of the component.